config/toggles.js

```diff
@@ -111,10 +111,6 @@ const shortTermToggles = {
   logInvalidHmsetValues: environmentVariableTrue(pe,
     'LOG_INVALID_HMSET_VALUES'),
 
-  // Enable GET from cache for /v1/subjects, /v1/subjects/{key}
-  getSubjectFromCache: environmentVariableTrue(pe,
-    'GET_SUBJECT_FROM_CACHE'),
-
```

Comment:
This means will not be using the GET_SUBJECT_FROM_CACHE flag and we will no longer get /v1/subjects and v1/subject/{key} from sample store correct?

Reply:
Just read your comment :)